src/streaming/controllers/PlaybackController.js

```diff
@@ -348,25 +371,23 @@ function PlaybackController() {
         }
     }
 
-    function getStartTimeFromUriParameters() {
+    function getStartTimeFromUriParameters(periodStart) {
```

Comment:
As this is not always periodStart since dvrWindow.start might be greater than periodStart I suggest we rename it to dvrWindowStart or something like that

Reply:
Yes you're right that it can be either a period start or a DVR window start.
What about rangeStart?